src/hexo/themes/sonarwhal/layout/partials/scan-result.hbs

```diff
@@ -15,7 +15,10 @@
             </header>
             <div class="section container">
                 <div class="scan-overview" data-id="{{id}}">
-                    <div class="scan-overview__header">SCANNED URL: <a href="{{url}}">{{url}}</a></div>
+                    <div class="scan-overview__header">
+                        <p>SCANNED URL: <a href="{{url}}">{{url}}</a></p>
+                        <p class="scan-overview__status" id="status-analyzing">Analyzing</p>
```

Comment:
It shouldn't be `Analyzing` if the users goes to a page that already has the results. It should be changed in the server side.